php_phongo.c

```diff
@@ -2445,6 +2445,12 @@ PHP_MINIT_FUNCTION(mongodb)
 
 	/* Initialize libmongoc */
 	mongoc_init();
+
+#ifdef MONGOC_EXPERIMENTAL_FEATURES
```

Comment:
How is this supposed to work with Decimal128? I expect all of the Decimal128 code would need to be wrapped in `MONGOC_EXPERIMENTAL_FEATURES` in that case.

@derickr and I spoke about this earlier in the week and I don't see any reason to release 1.2.0 until these features are properly available in libmongoc 1.5.0, which I expect you will need to have ready to go when MongoDB 3.2 ships.